app/controllers/places_controller.rb

```diff
@@ -55,6 +54,10 @@ def show
   end
 
 protected
+  def error_400(e)
+    error_message = e.message.gsub("MapitApi::", "").camelize(:lower)
+    render status: 400, json: {"error" => "#{error_message}"}
```

Comment:
Minor style guide notes:
I think this could be changed to `{ error: "#{error_message}" }`
That would use the preferred Ruby 1.9 syntax for hashes and adds spaces after `{` and before `}`

https://github.com/alphagov/styleguides/blob/master/ruby.md#general
https://github.com/alphagov/styleguides/blob/master/ruby.md#collections